### PR TITLE
fix: jobs undefined

### PR DESCRIPTION
**Pull request by BiswaViraj**

### CODE REVIEW

**Review by scopsy — APPROVED** [review]

**Comment by scopsy** [comment]
@BiswaViraj in what cases jobs can be undefined? Should we fallback to an empty array in that case? Also find it strange that a job can have undefined value in it.

**Comment by BiswaViraj** [comment]
@scopsy , Not sure how job was undefined, 
adding a fallback array as well

**Comment by scopsy** [comment]
@BiswaViraj could you do a bit of code investigation to try to find potential paths this could happen in? Maybe we have some other more deep problem we need to fix here.

**Comment by BiswaViraj** [comment]
@scopsy , Yup, I'm on it

File: apps/web/src/pages/activities/hooks/useNotificationStatus.ts [hunk 1]
```diff
@@ -5,8 +5,9 @@ export const useNotificationStatus = (notification: any): JobStatusEnum => {
   const [status, setStatus] = useState(JobStatusEnum.PENDING);
 
   useEffect(() => {
-    const result = notification?.jobs
```
Comment (scopsy) [inline apps/web/src/pages/activities/hooks/useNotificationStatus.ts:8]: I think this should happen in the backend, so the API actually returns an array not undefined here

**Pull request merged**